integrated_channels/utils.py

```diff
@@ -501,3 +508,38 @@ def get_enterprise_client_by_channel_code(channel_code):
         'canvas': CanvasAPIClient,
     }
     return _enterprise_client_model_by_channel_code[channel_code]
+
+
+def store_api_call(
+    enterprise_customer,
+    enterprise_customer_configuration_id,
+    endpoint,
+    payload,
+    time_taken,
+    status_code,
+    response_body,
+):
+    """
+    Creates new record in CornerstoneAPIRequestLogs table.
+    """
+    try:
+        integrated_channel_request_log_model().objects.create(
+            enterprise_customer=enterprise_customer,
+            enterprise_customer_configuration_id=enterprise_customer_configuration_id,
+            endpoint=endpoint,
+            payload=payload,
+            time_taken=time_taken,
+            status_code=status_code,
+            response_body=response_body,
+        )
+    except Exception as e:  # pylint: disable=broad-except
+        LOGGER.error(
+            f"store_api_call raised error while storing API call: {e}"
+            f"enterprise_customer={enterprise_customer}"
+            f"enterprise_customer_configuration_id={enterprise_customer_configuration_id},"
+            f"endpoint={endpoint}"
+            f"payload={payload}"
+            f"time_taken={time_taken}"
+            f"status_code={status_code}"
+            f"response_body={response_body}"
+        )
```

Comment:
I think object initialization functionality can be included within class, just like we do in the constructor (init). I believe this method can be added into the model itself as a class method. https://stackoverflow.com/questions/10586787/when-should-i-use-classmethod-and-when-def-methodself